### ISSUE TITLE
Analog tester gave reverse inputs

### ISSUE THREAD

**Issue by Ejoty123** [hrydgard/ppsspp#9903]
While mapping my Gamecube controller through a Mayflash adapter using x360ce [Yes, quite a few hoops to jump through], I was testing my left thumbstick with the Analog Tester in the controls section. The y axis was reversed. So I thought "If I map it in reverse, it should work fine. It still didn't work fine. Eventually I realized this was just an issue of the analog tester itself, and not the actual mapping, as when I booted up a game, up and down responded properly.
[Video of what I'm referring to, with Padpyght to show intended inputs](https://www.youtube.com/watch?v=piRgUb0--Mg&feature=youtu.be)

TL;DR whether the fault of x360ce or the mayflash 4-port adapter's drivers, PPSSPP's Analog Tester gave me a reversed Y-axis even though it worked properly in-game.

**Comment by hrydgard** [comment]
Known issue, already reported in https://github.com/hrydgard/ppsspp/issues/9300 . Closing as duplicate.
